examples/abi/src/lib.rs

```diff
@@ -44,3 +44,44 @@ impl Adder {
 fn sum_pair(a: &Pair, b: &Pair) -> Pair {
     Pair(a.0 + b.0, a.1 + b.1)
 }
+
+#[cfg(all(test, not(target_arch = "wasm32")))]
+mod tests {
+    use near_abi::*;
+    use tokio::fs;
+    use workspaces::prelude::*;
+
+    #[ignore]
```

Comment:
We are ignoring this because `cargo-near` is unavailable in `contract-builder`, right? We should create an issue to not forget to update this once we release `cargo-near` 0.1.0. I can create one if you want.

Reply:
yes, please.

Reply:
Done: https://github.com/near/near-sdk-rs/issues/903